src/processPattern.js

```diff
@@ -90,6 +90,21 @@ export default function processPattern(globalRef, pattern) {
             logger.debug(`${glob} doesn't match ${file.relativeFrom}`);
           }
 
+          // Run ignore functions
+          for (let i = 0; i < pattern.allIgnoreFunctions.length; i++) {
```

Comment:
Use `forEach`

Reply:
i've chosen the for loop, as i need to return the resolved promise, if one pattern matches. can I leave it like that for this use case?